crates/client/assets/schema.sdl

```diff
@@ -762,6 +762,10 @@ type Mutation {
 	"""
 	dryRun(txs: [HexString!]!, utxoValidation: Boolean, gasPrice: U64, blockHeight: U32): [DryRunTransactionExecutionStatus!]!
 	"""
+	Get execution trace for an already-executed transaction.
+	"""
+	storageReadReplay(height: U32!): [StorageReadReplayEvent!]!
```

Comment:
Why is it a mutation query?

Reply:
Ok I'm just going to change this to be a query. Created a follow-up to move dryRun over as well. https://github.com/FuelLabs/fuel-core/issues/2717